Content.Shared/Storage/Components/ItemCounterComponent.cs

```diff
@@ -20,7 +20,7 @@ namespace Content.Shared.Storage.Components
     public sealed partial class ItemCounterComponent : Component
     {
         [DataField("count", required: true)]
-        public EntityWhitelist Count { get; set; } = default!;
```

Comment:
Should this not be required anymore then?

Reply:
It probably doesn't need to be anymore. Though making it required makes sure that users actually want to count every item and not just forgot to add a whitelist. Most usages want to use a whitelist.

Reply:
remove funny get set as well and remove "count" :trollface: